extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php

```diff
@@ -1,14 +1,32 @@
 <?php
 
-class PortableInfoboxRenderServiceTest extends PHPUnit_Framework_TestCase {
+class PortableInfoboxRenderServiceTest extends WikiaBaseTest {
 	private $infoboxRenderService;
 
 	protected function setUp() {
 		$this->setupFile = dirname( __FILE__ ) . '/../PortableInfobox.setup.php';
 		parent::setUp();
 
-		$mock = $this->getMock( 'PortableInfoboxRenderService', [ 'getThumbnailUrl' ] );
-		$mock->expects( $this->any() )->method( 'getThumbnailUrl' )->will( $this->returnValue( 'http://image.jpg' ) );
+		$mockThumbnailImage = $this->getMockBuilder( 'ThumbnailImage' )
```

Comment:
In this case the getThumbnail will return false (simply logic is used there, didn't want to dig deeper in the thumbnailer error throwing functionality as it is not the subject of this test)